shared/middleware/helpers/storage_id.rb

```diff
@@ -8,7 +8,7 @@ def create_storage_id_cookie
   response.set_cookie(storage_id_cookie_name, {
     value:CGI.escape(storage_encrypt_id(storage_id)),
     domain:".#{site}",
-    path:'/v3/apps',
+    path:'/',
```

Comment:
If we wait for the client to call /v3/apps, we would have to jump through some hoops to make the call return synchronously, without freezing up the whole page. So it's not ideal. 

I suppose we could let the cookie through only for the small number of applab free play levels, but that wouldn't really buy us much unless it worked for share links to those levels too. So unless there's something I'm missing, I'm willing to believe that appOptions is a wrong path and focus on a client-side solution.